core/json/src/test/scala/net/liftweb/json/ExtractionBugs.scala

```diff
@@ -122,4 +122,50 @@ object ExtractionBugs extends Specification {
     val deserialized = parse(serialized).extract[Holder2]
     deserialized must_== holder
   }
+
+  "deserialize list of homogonous tuples w/ experimental tuples enabled" in {
```

Comment:
Given the enabled/disabled tests are identical here, I wouldn't be opposed to collapsing them into a single test---but don't care too much.